Template/Pages/Youth with a Mission.html

```diff
@@ -5,5 +5,5 @@
 [*
 <p>Είμαστε...</p>
 */*
-<p>We are...</p>
+<p>Youth With A Mission is a global movement of Christians from many cultures, age groups, and Christian traditions, dedicated to serving Jesus throughout the world. Also known as YWAM (pronounced “WHY-wham”), we unite in a common purpose to know God and to make Him known. Youth With A Mission is a truly global movement. We have over 1,100 locations in over 180 nations, and new teams are constantly being pioneered. Our centers reflect the culture they are in, and they often have multicultural staff. We are called to love, worship, and obey God, to love and serve His Body, the Church, and to present the whole gospel for the whole person throughout the whole world. We do this through evangelism, training, and mercy ministries.</p>
```

Comment:
Are we using British (‘...’) or American (“...”) quotation marks? The documents are labelled as British English, but we can switch everything if needed. As long as we are consistent.

[Commas aren’t thousands separators anymore](https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping).

```suggestion
<p><em>Youth With A Mission</em> is a global movement of Christians from many cultures, age groups, and Christian traditions, dedicated to serving Jesus throughout the world. Also known as YWAM (pronounced ‘WHY-wham’), we unite in a common purpose to know God and to make Him known. Youth With A Mission is a truly global movement. We have over 1100 locations in over 180 nations, and new teams are constantly being pioneered. Our centers reflect the culture they are in, and they often have multicultural staff. We are called to love, worship, and obey God, to love and serve His Body, the Church, and to present the whole gospel for the whole person throughout the whole world. We do this through evangelism, training, and mercy ministries.</p>
```

I don’t know what to do with “WHY‐wham”. It’s not even correct. It implies ˈʍaɪ̯.ʍɑːm when it’s really ˈwaɪ̯wæm.

Though American ears do not distinguish between the two, they are very different to the rest of the English‐speaking world. *whither* and *wither* aren’t homophones. You can also ask Catherine whether *trap* and *bath* have the same vowel—you’ll find yourselves disagreeing.

I *think* ˈwaɪ̯wæm is most‐correct for everyone, but I’m not sure if it is still useful written like that.

Maybe just removing the pronunciation note would be best? It’s not particularly important anyway.